commit 5098d0323852c22fd5fa78fc8c04e33b4410cd4a
Author: Juan RP <xtraeme@gmail.com>
Date:   Thu Jun 20 19:26:43 2019 +0200

    mk/prog.mk: set proper target for bins.
    
    Fixes build failure in aarch64-musl and cross compilation...
    weird it didn't fail on native.

diff --git a/mk/prog.mk b/mk/prog.mk
index 2d99ff55..007a1cac 100644
--- mk/prog.mk
+++ mk/prog.mk
@@ -46,12 +46,12 @@ endif
 	@printf " [CC]\t\t$@\n"
 	${SILENT}$(CC) $(CPPFLAGS) $(CFLAGS) $(EXTRA_CFLAGS) -c $<
 
-$(BIN).static: $(OBJS) $(STATIC_LIBS)
+$(BIN).static: $(OBJS) $(TOPDIR)/lib/libxbps.a
 	@printf " [CCLD]\t\t$@\n"
 	${SILENT}$(CC) -static $(OBJS) $(CPPFLAGS) -L$(TOPDIR)/lib \
 		$(CFLAGS) $(LDFLAGS) $(PROG_LDFLAGS) $(STATIC_LIBS) -o $@
 
-$(BIN): $(OBJS)
+$(BIN): $(OBJS) $(TOPDIR)/lib/libxbps.so
 	@printf " [CCLD]\t\t$@\n"
 	${SILENT}$(CC) $^ $(CPPFLAGS) -L$(TOPDIR)/lib \
 		$(CFLAGS) $(PROG_CFLAGS) $(LDFLAGS) $(PROG_LDFLAGS) \
